### DISCUSSION TITLE
Exploring why native executables produced with Mandrel 23.0 are bigger than those produced with Mandrel 22.3

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#36832] [Quarkus Blog/Website]
# Exploring why native executables produced with Mandrel 23.0 are bigger than those produced with Mandrel 22.3

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/mandrel-23-0-image-size-increase/

**Comment by bulldog98** [comment]
Super interesting read, into some the internals of why native compilation increases in size. I enjoyed the detailed explanation for the changes.

**Comment by roberttoyonaga** [comment]
Great presentation of the investigation! I like that you walked through your approach step by step and explained the resources/tools used.
